user_documentation_cf_cli_plugin.md

```diff
@@ -179,6 +185,66 @@ Restore has been aborted for the instance name: [SERVICE\_INSTANCE\_NAME]
 
 **Additional note:** The successful execution of this command means the abort process was initiated. Theprocess of aborting the backup again takes some time to complete. For the convenience of the user, the abort process too runs in the background. If you wish to know the progress and/or the state of the backup, you can use the &quot;cf service SERVICE\_INSTANCE\_NAME&quot; command.
 
+### Listing backups for deleted service instances:
+
+**Command:** cf list-backup SERVICE\_INSTANCE\_NAME --deleted
+
+**Usage:** This command is used to fetch backups for deleted service instances. You need to provide the name of the service-instance as the parameter. Upon successful execution of the command, the plugin will print all the backups taken for the deleted service instance.
+
+**Expected Output:**
+
+Getting the list of  backups in the org [ORG_NAME] / space [SPACE_NAME] / service instance [SERVICE\_INSTANCE\_NAME] ...
+
+OK
+
+[List of backups]
+
+**Example:**
+
+<img src="https://github.com/SAP/service-fabrik-cli-plugin/blob/master/images/plugin_screenshot4.png">
+
+**Additional note:** The successful execution of this command will return all the backups releated to the deleted service instance [SERVICE\_INSTANCE\_NAME]. 
+
+### Listing backups for deleted service instances using instance guid:
```

Comment:
Just to keep inline with above,
"Listing all backups of a service-instance by instance guid"